### PR TITLE
Test editing updating clear functions

### PR DESCRIPTION
**Pull request by hiromataba**
In this milestone, we have testes the drag and drop, editing,clear all functions

### CODE REVIEW

File: test/dragAndDrop.test.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import dragAndDrop from './__mocks__/dragAndDropMock';
+import './__mocks__/localStorageMock';
+
+describe('dragAndDrop', () => {
+  it('should throw an error if no task is given', () => {
+    expect(() => {
+      dragAndDrop.editTask();
+    }).toThrow();
+  });
+
```
Comment (mimipeshy) [inline test/dragAndDrop.test.js:10]: - Great job on the edge cases 👍🏾 Kindly add test to check if a user can update a task

File: test/dragAndDrop.test.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  it('should throw an error if no item is given', () => {
+    expect(() => {
+      dragAndDrop.clearCompleted();
+    }).toThrow();
+  });
+
```
Comment (mimipeshy) [inline test/dragAndDrop.test.js:16]: - [ ] Kindly add tests to check for the clear completed method. Good job on the edge cases too 👍🏾

File: test/dragAndDrop.test.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  it('should throw an error if no item is given', () => {
+    expect(() => {
+      dragAndDrop.update();
+    }).toThrow();
+  });
+
```
Comment (mimipeshy) [inline test/dragAndDrop.test.js]: - [ ] Good job on the edge cases 👍🏾 Kindly add tests for updating an item's index value upon drag/drop actions.

**Review by mimipeshy — CHANGES_REQUESTED** [review]
## Hi @hiromataba and @vladanvi99  👋🏾 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
#### Highlights
- good GitHub workflow ✔️ 
- not forming any linter errors ✔️
- Good job on testing negative scenarios ✔️  

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Renaming your branches in a simpler way `testing-p2` is much simpler than the current branch name.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Review by ZawadiSumba66 — APPROVED** [review]
### Hi Team 😃 , 
## STATUS: APPROVED ✔️ 
You have done well implementing the changes requested by the previous reviewer. 💯 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
#### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
